src/extensions/expokit/package.lisp

```diff
@@ -1,16 +1,13 @@
-;;;; transcendental/package.lisp
-;;;;
-;;;; Author: Robert Smith
+(defpackage #:magicl.expokit-cffi
+  (:use)
```

Comment:
Remove?

Reply:
You can't remove this. This explicitly says not to use any packages, including `:cl`. By removing it, it means `defpackage` can "choose" what default packages to use IIRC.